crates/scroll/payload/src/builder.rs

```diff
@@ -35,8 +35,6 @@ use revm::context::{Block, BlockEnv};
 use scroll_alloy_hardforks::ScrollHardforks;
 use std::{boxed::Box, sync::Arc, vec, vec::Vec};
 
-const SCROLL_GAS_LIMIT_10M: u64 = 10_000_000;
```

Comment:
I realized that I forgot to remove this in a previous PR, so adding this here to speed things up